azure-iot-device/azure/iot/device/common/pipeline/pipeline_stages_base.py

```diff
@@ -303,17 +371,20 @@ def _block(self, op):
     def _unblock(self, op, error):
         """
         Unblock this stage after the connection is complete.  This also means
-        releasing all the queued up operations that we were waiting for the
-        connect operation to complete.
+        releasing all the operations that were queued up.
         """
         logger.info("{}({}): disabling block and releasing queued ops.".format(self.name, op.name))
```

Comment:
>disabling block [](start = 29, length = 15)

`unblocking` will sound better i think

Reply:
+1

---
In reply to: [311821517](https://github.com/Azure/azure-iot-sdk-python-preview/pull/160#discussion_r311821517) [](ancestors = 311821517)